src/components/components/ebay-tooltip-overlay/template.marko

```diff
@@ -0,0 +1,34 @@
+<div
+    class=`${data.type}__overlay`
+    role=(data.type === 'tourtip'?'region':'tooltip')
+    aria-labelledby=(data.type === 'tourtip'?`tourtip-label-${widget.id}`:'')
```

Comment:
Similar to the above template literal comment, there are a few of these that should probably change.